tests/acceptance/content.feature

```diff
@@ -0,0 +1,18 @@
+Feature: content
+  In order to manage content article in the web
+  As an owner
+  I need to create content article
+
+  Background:
+    Given Joomla CMS is installed
+    When Login into Joomla administrator with username "puneet" and password "1234"
+    Then I see administrator dashboard
+
+  Scenario: Create an Article
+    Given There is a Add Content link
+    When I fill mandatory fields for creating article
+      | field   | value                    |
+      | title   | My_Article               |
+      | content | This is my First Article |
```

Comment:
very cool!

But I would only use tables when we want to repeat the same test with different values.

Fore example: create 10 articles:

- article1 | content1
- article2 | content2
- article3 | content3

see:
![screen shot 2016-06-10 at 11 18 19](https://cloud.githubusercontent.com/assets/1375475/15959883/419b25c6-2efd-11e6-97e3-1a585ab618c1.png)